src/print.c

```diff
@@ -245,6 +259,183 @@ launch_preview (const char *filename,
   return retval;
 }
 
+static int
+pdf_get_actions_page (char        *filename,
+                      PDF_ACTIONS  act,
+                      int          page)
+{
+  GSubprocess *process;
+  GInputStream *stream;
+  GPtrArray *args = g_ptr_array_new_full(8, g_free);
+  char buffer[50] = { 0 };
+
+  g_ptr_array_add(args, LIBEXECDIR "/pdftoraw");
```

Comment:
You need to `g_strdup()` ever arg passed to `g_ptr_array_add()`.

Reply:
Fixed by 9cb421f